wordpress-mobile/WordPress-Aztec-iOS
# Add support for adding image to DOM.

Add support for adding images to the HTML/DOM.

How to test:

 - Start the demo app
 - Go to editor demo
 - Add an image to the editor
 - Check the HTML view to see if the img tag was added
 - Go back and check if the image is still there
 - Remove the image on the visual mode
 - Check the HTML
 - Repeat the process but now remove image on the HTML mode.
